### PR TITLE
Component: Lists  completed

### PR DESCRIPTION
**Pull request by yogesh-**
completed Lists component which contains normal lists and notification list.

image

image

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
code looks clean with proper naming with good UI

File: documentation.html [hunk 1]
```diff
@@ -1276,6 +1279,126 @@ <h1 class="h1-style">Snackbar</h1>
...
+
+                    <!-- simple list -->
+                    <h2 class="h2-style">Simple List</h2>
+                    <p class="p-style">Simple list is a vertical index of items. The items can be text,images or
+                        hyperlinks in various combinations.<br>
+
+                        Use class of <span>simple-list</span> on the main class and class of <span>list-items</span> for
+                        the items in the list.
+
+
```
Comment (akash-1318) [inline documentation.html]: remove these extra line spacing.

**Review by akash-1318 — COMMENTED** [review]
Just added a simple suggestion besides that everything looks good with proper formatting and naming convention.

File: documentation.html (hunk 1 above)
Comment (yogesh-) [reply documentation.html]: done

**Pull request merged**